### PR TITLE
Landscape/floor texture + antialiasing (mipmap)

### PR DESCRIPTION
**Pull request by rtr-dnd**
## Context
There was really ugly aliasing issues with backgrounds.

## Summary
- Landscapes now uses texture insted of drawing in fragment shader.
- Vertex info of floor was changed to make use of mipmaps.

## How to check behavior
Run zennist.
This requires mipmap support on zigen/zen/zen-remote/zukou.

### CODE REVIEW

**Comment by rtr-dnd** [comment]
Check is failing because this requires zukou 0.0.3 (mipmap support).

**Comment by Aki-7** [comment]
https://github.com/zigen-project/zukou/pull/15 was merged!

File: src/floor.cc [hunk 1]
```diff
@@ -4,6 +4,7 @@
 #include <unistd.h>
 
 #include <cstring>
+#include <iostream>
```
Comment (shierote) [inline src/floor.cc:7]: [nits] unused header. (maybe for debugging?)

**Review by shierote — APPROVED** [review]
LGTM!
This works on my env as well!

**Pull request merged**